db/silkworm/common/log.hpp

```diff
@@ -36,7 +36,7 @@ namespace silkworm {
 #define SILKWORM_LOG_VERBOSITY(level_) (silkworm::log_verbosity_ = (level_))
 
 // available verbosity levels
-enum LogLevels { LogTrace, LogDebug, LogInfo, LogWarn, LogError, LogCritical, LogNone };
+enum class LogLevels { LogTrace, LogDebug, LogInfo, LogWarn, LogError, LogCritical, LogNone };
```

Comment:
I'd rename to `LogLevel` and remove the `Log` prefix from the values so that we could say e.g.
```
SILKWORM_LOG(LogLevel::Info) << "ETL Load Progress "
```
rather than
```
SILKWORM_LOG(LogLevels::LogInfo) << "ETL Load Progress "
```

Reply:
Sounds better. Agree